sys/dev/netmap/netmap.c

```diff
@@ -3079,15 +3150,26 @@ netmap_common_irq(struct ifnet *ifp, u_int q, u_int *work_done)
 	}
 
 	if (q >= nma_get_nrings(na, t))
-		return;	// not a physical queue
+		return 0; // not a physical queue
```

Comment:
If you are using the netmap patched ixgbe drivers, this will cause the driver to fall back to regular processing.